api/problem/create.py

```
@@ -1,40 +1,45 @@
+# FUNCTION -> DECLARATION
+@router.post("/api/problem/create")
+async def output(request: Request, response: Response) -> JSONResponse:
+    # DECLARATION -> EXTENSIONS
+    from website.extensions import database, post, time
+    await asyncio.gather(
+        database.init(request, response),
+        post.init(request, response),
+        time.init(request, response)
+    )
+    # DECLARATION -> ARGUMENT CHECKS
+    if not post.checks: raise SUG.ERR[0]
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)